app/views/inthenewscommentaries/home.html.erb

```diff
@@ -11,7 +11,7 @@
   </div>
 
   <div class="col-md-9">
-    <div>
+    <div class="row">
```

Comment:
We do actually want this to be just `<div>`, apologies!

Reply:
`<div class="row">` inside a `<div class="col-*-*>` does something weird in Bootstrap 3, just trying to fix that